File: Easy/JavaScript/455. Assign Cookies.js
var findContentChildren = function(g, s) {
    let count = 0;
    let i = 0;
    g.sort((a, b) => a - b);
    s.sort((a, b) => a - b);

    for(let j = 0; j < g.length; j++){
        while(i < s.length && s[i] < g[j])
            i++;
        if(i == s.length)
            break;
        i++;
        count++;
    }

    return count;
};

File: Easy/Python/455. Assign Cookies.py
class Solution(object):
    def findContentChildren(self, g, s):
        count = 0
        i = 0
        g.sort()
        s.sort()

        for j in range(len(g)):
            while i < len(s) and s[i] < g[j]:
                i += 1
            if i == len(s):
                break
            i += 1
            count += 1
        
        return count
